### ISSUE TITLE
Fix Locals and Watch List

### ISSUE THREAD

**Issue by danwalmsley** [VitalElement/AvalonStudio#94]
- [x] waiting for https://github.com/AvaloniaUI/Avalonia/issues/604
- [x] Ensure when debugger is closed, watch list no longer tries to access debugger and crashes.
- [x] Fix stability issues with expanding.
- [x] Fix re-parsing tree when an object when out of scope and then comes back in scope. 
- [ ] Implement editing of values.
- [ ] Display format (may already work)

**Comment by danwalmsley** [comment]
the avalonia bug that this was waiting on has now closed.
